app/models/hackbot/interactions/sql.rb

```diff
@@ -0,0 +1,133 @@
+class Executer
+  def execute(query)
```

Comment:
@zachlatta We could have a button after the query results are posted that shares that makes the result not ephemeral. There's a few ways we can make this command work, but I'm partial to adding slash commands first.